src/templates/gwfcore/gwfcore-batch.template.yaml

```diff
@@ -257,6 +257,12 @@ Resources:
         - Order: 2
           ComputeEnvironment: !Ref OnDemandComputeEnv
 
+  ContainerInstanceLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: "/aws/ecs/container-instance"
```

Comment:
Possible to add the `Namespace` to the log-group? GWFcore can be installed multiple times. Adding the namespace will help isolate instances in that scenario.